frontend/src/libs/helpers/build-query-string/build-query-string.helper.ts

```diff
@@ -0,0 +1,15 @@
+const buildQueryString = <T extends Record<string, string | number | boolean>>(
+  queryParameters: T | null,
+): string => {
+  if (queryParameters === null) {
+    return '';
+  }
+
+  const queryString = Object.keys(queryParameters)
+    .map((key) => `${key}=${queryParameters[key].toString()}`)
+    .join('&');
```

Comment:
we have already query service implemented, can we reuse setQueryParameters from it?

Reply:
We only have the useQueryParameters hook, which contains this setQueryParameters method, which I cannot use in the trucks-api. And in order not to manually write the query string to all api every time, I added this small builder.